rectools/dataset/dataset.py

```diff
@@ -245,3 +246,67 @@ def get_raw_interactions(self, include_weight: bool = True, include_datetime: bo
         pd.DataFrame
         """
         return self.interactions.to_external(self.user_id_map, self.item_id_map, include_weight, include_datetime)
+
+    def filter_on_interactions_df_row_indexes(
+        self,
+        row_indexes_to_keep: np.ndarray,
```

Comment:
Since we refused python 3.7 support, let's start adding types for numpy arrays. Please specify here that this is array of integers

Reply:
Decided to do it later